sw/device/lib/testing/pwrmgr_testutils.c

```diff
@@ -33,3 +33,20 @@ bool pwrmgr_testutils_is_wakeup_reason(const dif_pwrmgr_t *pwrmgr,
           wakeup_reason.types == kDifPwrmgrWakeupTypeRequest) &&
          wakeup_reason.request_sources == reasons;
 }
+
+void pwrmgr_testutils_enable_usb_clock_in_active_state(
```

Comment:
Like the function below, this incurs a full cdc sync, which is a bit undesirable.

Reply:
removed.